src/app/api/events/[id]/rsvp/route.ts

```
@@ -36,14 +36,34 @@ export async function POST(
         // 2. Check score
         if (verification.score >= 0.7) {
             // Approve normally
-            const { error: rsvpError } = await supabase
-                .from('event_registrations')
-                .insert({ event_id: eventId, user_id: userId, status: 'confirmed' });
+const { data, error: rsvpError } = await supabase.rpc('join_event_or_waitlist', {
```

ESLint

[error] Replace `const·{·data,·error:·rsvpError·}·=·await·supabase.rpc('join_event_or_waitlist'` with `······const·{·data,·error:·rsvpError·}·=·await·supabase.rpc("join_event_or_waitlist"`

(prettier/prettier)